spec/sanity_check_spec.rb

```diff
@@ -34,7 +34,7 @@ def with_clean_env
     end
   end
 
-  it "passes when libraries are required" do
+  it "passes when libraries are required", unless: RSpec::Support::Ruby.jruby? do
```

Comment:
Do you think we should revert that change when gem/json-jruby warning will be fixed?

Reply:
Yes probably.